### DISCUSSION TITLE
C/C++ of pytorch_geometric

### DISCUSSION THREAD

**Discussion by Haoyu-ha** [pyg-team/pytorch_geometric#4186] [Q&A]
I refer to the official documents to convert the model to Torchscript, but libtorch failed to load the model? The error is “Unknown builtin op: torch_sparse::ptr2ind. Could not find any similar ops to torch_sparse::ptr2ind. This op may not exist or may not be currently supported in TorchScript”. How can I solve this problem?

**Comment by rusty1s** [comment]
Please refer to this issue: https://github.com/pyg-team/pytorch_geometric/issues/1718. We will work on a fix.
